src/compas_timber/ghpython/components/CT_Joint_Options_LButt/code.py

```diff
@@ -15,6 +15,6 @@ def RunScript(self, small_beam_butts, modify_cross, reject_i):
         if reject_i is not None:
             args["reject_i"] = reject_i
 
-        options = JointOptions(LButtJoint, **args)
+        options = JointOptions(LButtJoint, ["main_beam", "cross_beam"], **args)
```

Comment:
Sorry to nitpick but since these names end up on the components should we not use CamelCase here as we usually do?

Reply:
done